awx/settings/defaults.py

```diff
@@ -114,6 +114,7 @@
 MEDIA_URL = '/media/'
 
 LOGIN_URL = '/api/login/'
+LOGOUT_ALLOWED_HOSTS = os.getenv('LOGOUT_ALLOWED_HOSTS').split(",") if os.getenv('LOGOUT_ALLOWED_HOSTS') else []
```

Comment:
```suggestion
LOGOUT_ALLOWED_HOSTS = []
```

Reply:
although it should be possible for us to read in from env var we don't seem to have any precedent around doing this 

using awx.spec.extra_settings to inject extra setting is the current established pattern